repository-data/webfiles/src/main/resources/site/less/demo.less

```diff
@@ -1,303 +1,458 @@
+@max-width: 1170px;
+@color-primary: #005EB8;
+@color-secondary: #00376D;
+@color-grey-dark: #3E4F59;
+@color-grey-light: #DEE2E5;
+
+
+
+@base-spacing-unit: 20px;
+@half-spacing-unit: @base-spacing-unit/2;
+@quarter-spacing-unit: @base-spacing-unit/4;
+@double-spacing-unit: @base-spacing-unit*2;
+
+@base-radius: 5px;
 
 
-/*
-<div class="container">
-</div>
- */
 .container {
-    width: 1200px;
+    width: @max-width;
     margin: 0 auto;
 }
 
-.container-side {
-    width: 20%;
-    float: left;
-    display: inline-block;
+
+
+
+/* TEXT */
+/* ########################################################## */
+
+body {
+  font-family:"FrutigerLTW01-55Roman", Arial;
 }
 
-.container-main {
-    display: inline-block;
-    width: 78%;
+h1, h2, h3, h4, h5, h6 {
+    margin-top: 0;
+    margin-bottom: @half-spacing-unit;
+    line-height: 1.3;
+    color: @color-grey-dark;
+    font-family:"FrutigerLTW01-55Roman", Arial;
+    font-weight: 100;
 }
 
-.container-main:first-child {
-    width: 100%;
+h1 {
+  font-size:36px;
+}
+h2 {
+  font-size:33px;
+  font-family:"FrutigerLTW01-65Bold", Arial;
+}
+h3 {
+  //font-family:"FrutigerLTW01-65Bold", Arial;
 }
 
+p {
+  font-size:18px;
+  color: @color-grey-dark;
+  line-height:1.5;
+  margin-bottom:30px;
+}
 
-/**
-<ul class="container-top__menu">
-    <li>...</li>
-</ul>
- */
-.container-top__menu {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-    overflow: hidden;
+ul, li {
+  font-size:18px;
+  color: @color-grey-dark;
+  line-height:1.5;
 }
-.container-top__menu li {
-    float: left;
-    display: block;
-    text-align: center;
-    padding: 16px;
-    text-decoration: none;
+
+a {
+    color: inherit;
+    //text-decoration: none;
+    color: @color-primary;
 }
-.container-top__menu a {
-    text-decoration: none;
+
+.zeta {
+  font-size:16px;
 }
-.in-line-container {
-    display: inline-block;
+
+
+/* SPACING */
+/* ########################################################## */
+
+.flush {
+  margin:0px;
+}
+.flush--top {
+  margin-top:0px;
+}
+.flush--bottom {
+  margin-bottom:0px;
+}
+.flush--left {
+  margin-left:0px;
+}
+.flush--right {
+  margin-right:0px;
 }
 
-.column-half__left {
-    float: left;
-    clear: left;
-    width: 49%;
+
+.push--top {
+  margin-top:@base-spacing-unit;
+}
+.push-double--top {
+  margin-top:@double-spacing-unit;
+}
+.push-half--top {
+  margin-top:@half-spacing-unit;
 }
 
-.column-half__right {
-    float: right;
-    clear: right;
-    width: 49%;
+.push--bottom {
+  margin-bottom:@base-spacing-unit;
+}
+.push-double--bottom {
+  margin-bottom:@double-spacing-unit;
+}
+.push-half--bottom {
+  margin-bottom:@half-spacing-unit;
 }
 
 
-/*
-<a class="logo logo-nhs-digital">
-    text that will not be displayed
-</a>
-*/
-.logo {
-    display: block;
-    height: 100px;
-    width: 100px;
-    background-repeat: no-repeat;
-    text-indent: -9999em;
+
+
+
+
+/* HEADER */
+/* ########################################################## */
+
+.header {
```

Comment:
As discussed, can we add some qualifier to this, renaming it to something like 'top-header'?